receiptui/src/androidTest/java/com/hyperwallet/android/ui/receipt/ListPrepaidCardReceiptsTest.java

```diff
@@ -308,7 +338,10 @@ public void testListPrepaidCardReceipt_checkDateTextOnLocaleChange() {
         onView(withId(R.id.list_receipts)).check(
                 matches(atPosition(0, hasDescendant(withText("- 8,90")))));
         onView(withId(R.id.list_receipts)).check(
-                matches(atPosition(0, hasDescendant(withText("1. Juni 2019")))));
+                matches(atPosition(0,
+                        hasDescendant(withText(formatDateTime(mActivityTestRule.getActivity().getApplicationContext(),
+                                DateUtils.fromDateTimeString("2019-06-01T22:49:17").getTime(),
```

Comment:
I think we are providing localisation for Date & time only. Do we going to implement localisation for content also?
Because it may not be look good if we implement localisation for date & time only i think. @fmattos-hw @peter-joseph